app.rb

```diff
@@ -0,0 +1,226 @@
+require './student'
+require './teacher'
+require './book'
+require './rental'
+require 'io/console'
+
+class App # rubocop:disable Metrics/ClassLength
```

Comment:
- [ ] Well done @HenaryKhuraijam, you implemented the features very well, but as per the general `requirements` you shouldn't disable the `robocop` linters in any situation. Because `linters` are meant to help you to write **code** more efficiently and cleanly. Please remove the `linter` disable comment and try to `resolve` the `linters` errors.